adidas-ascii-logo/swift/README.md

```diff
@@ -0,0 +1,16 @@
+# adidas ASCII logo: JavaScript
+
+Implementation to resolve the [adidas ASCII logo challenge][adidas-ascii-logo] with Swift.
+
+## Requirements and dependencies
+
+[Swift/Swift Package Manger][nodejs] are required to work with the repository.
```

Comment:
I think we have to change `nodejs` to the link for `swift`'s homepage

Also, `are` -> `is`

Reply:
Done